en-US/firefox/features/sync.lang

```diff
@@ -0,0 +1,81 @@
+## NOTE: url for demo server is: https://www-dev.allizom.org/firefox/sync/
+## URL: https://www-dev.allizom.org/%LOCALE%/firefox/sync/
+
+
+# HTML page title
+;Use Bookmarks, Tabs and Passwords Across Devices | Firefox
+Use Bookmarks, Tabs and Passwords Across Devices | Firefox
+
+
+# HTML page description
+;Bring your Firefox bookmarks, tabs, or passwords wherever you go with Firefox Sync. Share between computers and mobile devices. Get Firefox now!
+Bring your Firefox bookmarks, tabs, or passwords wherever you go with Firefox Sync. Share between computers and mobile devices. Get Firefox now!
+
+
+;Browse uninterrupted with Firefox Sync
+Browse uninterrupted with Firefox Sync
+
+
+# Displayed only to Firefox user with 31 and later, signed in into Firefox Accounts
+;Sync across devices
+Sync across devices
+
+
+;With Firefox on all your devices, you can access bookmarks, tabs and passwords with one easy sign in.
+With Firefox on all your devices, you can access bookmarks, tabs and passwords with one easy sign in.
```

Comment:
This one should indeed be sign-in

Reply:
Meant to comment "sign-in" here. Looks like en-US has changed: https://github.com/mozilla/bedrock/pull/5099/files#diff-851f8fad4132c8cb87ce5e150faed4c3R41

Reply:
You're commenting an old version, it's sign-in for us too already